src/commands/dev/lib/builder-cache.ts

```diff
@@ -9,20 +9,19 @@ import cacheDirectory from 'cache-or-tmp-directory';
 import wait from '../../../util/output/wait';
 import { Output } from '../../../util/output';
 import { devDependencies as nowCliDeps } from '../../../../package.json';
-import { Builder, PACKAGE } from './types';
+import { Builder, BuilderWithPackage } from './types';
 import {
   NoBuilderCacheError,
   BuilderCacheCleanError
 } from '../../../util/errors-ts';
 
-import * as _staticBuilder from './static-builder';
-const staticBuilder: Builder = _staticBuilder;
-staticBuilder[PACKAGE] = {
-  version: 'built-in'
-};
+import * as staticBuilder from './static-builder';
 
-const localBuilders: { [key: string]: Builder } = {
-  '@now/static': staticBuilder
+const localBuilders: { [key: string]: BuilderWithPackage } = {
+  '@now/static': {
+    builder: Object.freeze(staticBuilder),
+    package: { version: 'built-in' }
```

Comment:
It's a bit strange to see 'vbuilt-in' in logs

```suggestion
    package: {
      version: `${require('../../../../package.json').version}[built-in]`
    }
```

Reply:
Ya, I meant to make this be an empty string so that the version doesn't get printed.